lib/libdvblinkremote/curlhttpclient.cpp

```diff
@@ -22,7 +22,7 @@
  ***************************************************************************/
 
 #include "curlhttpclient.h"
-#include <memory.h>
+#inclide <memory.h>
```

Comment:
This file is not even built at all. I will correct the syntax error and will do new pull request.